convex/subscriptions.ts

```
@@ -284,7 +283,9 @@ export const manageTeamOnSubscriptionChange = internalMutation({
       // Delete all team members
       const teamMembers = await ctx.db
         .query('team_members')
-        .filter((q) => q.eq(q.field('team_id'), oldSubscription.team_id!))
+        .withIndex('by_team_id', (q) =>
+          q.eq('team_id', oldSubscription.team_id!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)